src/Components/ResultsFilter/index.js

```diff
@@ -0,0 +1,6 @@
+import ResultsFilter from './ResultsFilter'
+import PerPageDropdown from './elements/PerPageDropdown'
+
+ResultsFilter.PerPageDropdown = PerPageDropdown
```

Comment:
Is the dependency necessary?

Reply:
Yeah I think so.  It should only ever be used within a <ResultsFilter /> which contains the pagination info and number of Results.